dao/searchDao.js

```diff
@@ -6,5 +6,6 @@ exports.get = function(filter, cb) {
         allFilters.classFilter(filter.class, db) + allFilters.seatFilter(filter.seat, db) +
         allFilters.departureDateFilter(filter.departureDateAfter, filter.departureDateBefore, db) + allFilters.arrivalDateFilter(filter.arrivalDateAfter, filter.arrivalDateBefore, db) +
         allFilters.departureLocationFilter(filter.departure_location, db) + allFilters.destinationLocationFilter(filter.destination_location, db) + ';';
+        // filter out reserver by default because we want tickets that have not been taken yet
```

Comment:
I would move this above line 5; it's describing the `WHERE` clause, not the `query` call.

Reply:
Yeah, but that would be a lot of variables and I am not sure if it is worth it. It might be because of the increase in readability.

Reply:
"This" meaning just the comment.

Reply:
Oh got it!